Code/Engine/GameEngine/GameApplication/Implementation/GameApplicationInit.cpp

```diff
@@ -234,7 +234,6 @@ const char* ezGameApplication::GetActiveRenderer()
 
 void ezGameApplication::Init_SetupGraphicsDevice()
 {
-#if EZ_ENABLED(EZ_PLATFORM_WINDOWS)
```

Comment:
But it only runs on Win + Linux, not on ALL platforms, no ?

Reply:
Currently win + linux are the only supported platforms, correct. But in theory we can support all platforms that we can run vulkan on (android, MacOS). I just wanted to avoid that the next person who adds a new platform has to change this place again. After all this function does not have any platform specific code. It only uses our abstracted interfaces. So I don't see any reason to make it platform specific.